platformio.ini

```diff
@@ -156,8 +156,19 @@ build_flags = ${env.build_flags}
 	-DOPENSHOCK_ESTOP_PIN=38
 	-DARDUINO_USB_CDC_ON_BOOT=1
 
-; TODO:
-; https://docs.platformio.org/en/latest/boards/espressif32/upesy_wroom.html;upesy-esp32-wroom-devkit
+; https://docs.platformio.org/en/stable/boards/espressif32/nodemcu-32s.html
+[env:NodeMCU-32S]
+board = nodemcu-32s
+custom_openshock.chip = ESP32
+build_flags = ${env.build_flags}
+	-DOPENSHOCK_LED_GPIO=2
+
+; https://docs.platformio.org/en/latest/boards/espressif32/upesy_wroom.html
+; untested, but compiles fine
```

Comment:
Why add support for it then?

Reply:
Someone left a TODO, hopefully they could test it. If not, I have tested NodeMCU-32S already and can confirm it works.

Reply:
seems like @hhvrc added that todo, why?

Reply:
I'm not sure, I'll omit uPesy until someone can test it